docs/reference/api/client/_index.md

```diff
@@ -1,8 +1,9 @@
 ---
 title: "Client (Kn)"
```

Comment:
This renders a blank page. I don't think we should publish a reference page that has no reference info (to avoid user frustration).